.github/workflows/dev-build-deploy.yml

```
@@ -33,29 +38,28 @@ jobs:
+        uses: appleboy/ssh-action@master
         with:
           host: ${{ secrets.SERVER_HOST }}
           username: ${{ secrets.SERVER_USERNAME }}
           password: ${{ secrets.SERVER_PASSWORD }}
           port: ${{ secrets.SERVER_PORT }}
           script: |
-            sudo docker-compose pull ${{ secrets.DOCKERHUB_USERNAME }}/aics-api:latest
-            sudo docker-compose up -d
\ No newline at end of file
+            echo "${{ secrets.SERVER_PASSWORD }}" | sudo -S docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} pull
+            echo "${{ secrets.SERVER_PASSWORD }}" | sudo -S docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} up -d
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)